src/renderer/src/routes/pages/Wallets/index.tsx

```diff
@@ -41,6 +49,45 @@ export const WalletsPage = () => {
     setIsReordering(false)
   }
 
+  const timeToDate = (unixTime: number): string => {
+    const date = new Date(unixTime * 1000) // Converte para milissegundos
+    return date.toLocaleDateString() // Use toLocaleDateString() para obter a representação da data
+  }
+
+  const timeToHour = (unixTime: number): string => {
+    const date = new Date(unixTime * 1000) // Converte para milissegundos
+    return date.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' }) // Use toLocaleTimeString() para obter a representação da hora
+  }
```

Comment:
What do you think about creating a new DateHelper e add these two functions there ?

Reply:
Don't forget to remove these comments